hpcgap/src/gvars.c

```diff
@@ -146,11 +160,16 @@ void UnlockGVars() {
 **  will return the value of <gvar>  after evaluating <gvar>-s expression, or
 **  0 if <gvar> was not an automatic variable.
 **
-**  'VAL_GVAR' is defined in the declaration part of this package as follows
-**
-#define VAL_GVAR(gvar)          (PtrGVars[GVAR_BUCKET(gvar)] \
-                                [GVAR_INDEX(gvar)-1])
 */
+#undef VAL_GVAR
+#define VAL_GVAR(gvar)          (PtrGVars[GVAR_BUCKET(gvar)] \
```

Comment:
Yes, it was more the redefine, now VAL_GVAR means something different in this file, and outside it, so people looking at the definition have to be careful.